paddle/fluid/framework/details/multi_devices_graph_pass.cc

```diff
@@ -932,6 +940,17 @@ int DistSSAGraphBuilder::CreateDistTrainOp(ir::Graph *result,
   return op_dev_id;
 }
 
+bool DistSSAGraphBuilder::IsEncoded(const std::string &p_name) const {
+  auto u_name = p_name + "__dgc_u__";
```

Comment:
Maybe we should use a function to get the `__dgc_u__ `, because it is also used in [here](https://github.com/PaddlePaddle/Paddle/pull/15841/files#diff-91ee3921323472bc3550bcf1709eb34fR703).